### PR TITLE
#92 formats

### PR DESCRIPTION
**Pull request by banukutlu**
Connected to #92

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (cdmo) [inline Gemfile]: Must have run rubocop with MRI? I have been wondering about leaving this fix in or not. Guess it doesn't hurt jruby.

Comment (banukutlu) [reply Gemfile]: oh yeah at some point I ran rubocop with MRI - I did not realize these - I don't know if it'd hurt to have them with jruby 🤔 but it did not give me any errors when I index the sample with jruby 🤔

Comment (cdmo) [reply Gemfile]: 👌

File: lib/marc_pub_date_processor.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
```
[the remark this answers is not in the document]
Comment (banukutlu) [reply lib/marc_pub_date_processor.rb]: yes exactly, just moved out of psulib_marc.rb, no other changes.

File: lib/traject/psulib_config.rb [hunk 3]
```diff
@@ -177,7 +167,34 @@
 ## Author sorting field
 to_field 'author_ssort', marc_sortable_author
 
-# Subject field(s):
+# Formats and Resources
```
[the remark this answers is not in the document]
Comment (banukutlu) [reply lib/traject/psulib_config.rb]: Yes correct

**Pull request merged**